src/components/styles/StyledWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+// components/StyledWrapper.tsx
+import React, { FC, ReactNode } from 'react';
+
+interface StyledWrapperProps {
+  children: ReactNode;
+}
+
+const StyledWrapper: FC<StyledWrapperProps> = ({ children }) => {
+
+  return (
+    <main className="page-padding flex min-h-screen flex-col">
```

Comment:
This could also be a class. Or are there benefits to have them as components?

Reply:
Actually, the components are not 100% ready yet.
And to be honest, I'm not entirely sure what I'm doing.

However,
I'm trying to implement some sorts of  style components
For the header, content, footer, and other parts of the application or pages, so they can be reused across all pages.

This way, in the future, if we want to change something, we can just update that component, and the changes will be reflected throughout the entire app.

Reply:
Alright I see, yeah wrapper components should probably just be a global style if the components only purpose is to style content. 

But since you say the components are not 100% ready we can keep it this way for now and maybe convert them to styles later if they don't gain more function 👍